### PR TITLE
94 consider restructuring the guides section

### LINKED ISSUE #94: Consider restructuring the Guides section
> instead of having our product as the parent page, we should focus on specific features. For example, we can have a page called "Staking" and then break it down into sub-pages like "Staking in Polkaswap” and "Staking on Mobile." It just makes more sense, you know? Let me know what you think.

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: src/id.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 SORA allows you to store profile information in order to identify and discover people across the network. It's convenient for users to use the identity and registrar features. A tutorial is provided below on how to set up an identity for yourself.
 
+## Via PolkadotJS UI
+
 ## Register an Identity
```
Comment (outoftardis) [inline src/id.md]: thus then should be level 3 heading, same for other sections that are within the "via polkadot" section 


```suggestion
### Register an Identity
```

**Review by outoftardis — APPROVED** [review]

File: .vitepress/config.ts [hunk 2]
```diff
@@ -97,36 +99,55 @@ export default defineConfig({
...
+          { text: "Connect Wallet", link: "polkaswap-connect-wallet" },
           {
-            text: "Assets",
-            link: "assets",
-            collapsed: true,
+            text: "Send & Receive",
+            link: "send-and-receive",
+          },
+          {
+            text: "Explore Ecosystem",
```
Comment (WRRicht3r) [inline .vitepress/config.ts]:
```suggestion
            text: "Explore the Ecosystem",
```

File: src/advanced-trading.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Advanced Trading
 
+## Theory
+
 To Polkaswap means to exchange (swap) tokens on [Polkaswap](https://polkaswap.io/), Polkaswap is a non-custodial, cross-chain AMM DEX protocol for swapping tokens, Polkaswap removes trusted intermediaries and provides the opportunity for faster trading, and Polkaswap also combines multiple liquidity sources under a common liquidity aggregation algorithm, operating completely on-chain, in a trustless and decentralized way.
 
 Polkaswap's technical design allows using different liquidity sources for trading. At the moment Polkaswap supports two liquidity sources:
```
Comment (WRRicht3r) [inline src/advanced-trading.md:7]: Soon we will have to add the XST platform

File: src/check-supply.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Check Supply
 
+## Via Polkaswap
+
 ::: info
 Please consider that you would see the **total supply** (this includes locked tokens and the circulating supply) of all tokens in the network and not only the **circulating** supply of the tokens.
```
Comment (WRRicht3r) [inline src/check-supply.md]:
```suggestion
Please be aware that you would see the **total supply** (this includes locked tokens and the circulating supply) of all tokens in the network and not only the **circulating** supply of the tokens.
```

File: src/fast-track-public-voting.md [hunk 5]
```diff
@@ -28,11 +30,11 @@ You should see the council motion in the “Governance > Council“ tab:
 
 <figure><img src="/.gitbook/assets/public-voting-view-council-motion.png" alt=""><figcaption></figcaption></figure>
 
-## Step 3: Ask SORA Council to Vote
+**Step 3**: Ask SORA Council to Vote
```
Comment (WRRicht3r) [inline src/fast-track-public-voting.md]:
```suggestion
**Step 3**: Ask the SORA Council to Vote
```

File: src/fast-track-public-voting.md [hunk 6]
```diff
@@ -56,11 +58,11 @@ You should see the technical committee motion in the “Governance / Tech. comm.
 
 <figure><img src="/.gitbook/assets/public-voting-create-technical-committee-motion.png" alt=""><figcaption></figcaption></figure>
 
-## Step 6: Ask SORA Technical Committee to Vote
+**Step 6**: Ask SORA Technical Committee to Vote
```
Comment (WRRicht3r) [inline src/fast-track-public-voting.md]:
```suggestion
**Step 6**: Ask the SORA Technical Committee to Vote
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**